packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -7,9 +7,12 @@ version: 5.2.7
 
 flutter:
   plugin:
-    androidPackage: io.flutter.plugins.urllauncher
-    iosPrefix: FLT
-    pluginClass: UrlLauncherPlugin
+    platforms:
+      android:
+        package: io.flutter.plugins.urllauncher
+        pluginClass: UrlLauncherPlugin
+      ios:
+        pluginClass: FLTUrlLauncherPlugin
 
 dependencies:
   flutter:
```

Comment:
No new dependency on `url_launcher_web` yet?

Reply:
Good point, I guess we might as well do it in this PR as it is already gated on a new enough Flutter

Reply:
done